phpstan.neon.dist

```diff
@@ -24,6 +24,11 @@ parameters:
             message: '~^Method Doctrine\\Migrations\\Tests\\Stub\\DoctrineRegistry::getService\(\) should return Doctrine\\Persistence\\ObjectManager but returns Doctrine\\DBAL\\Connection\|Doctrine\\ORM\\EntityManager~'
             path: tests/Doctrine/Migrations/Tests/Stub/DoctrineRegistry.php
 
+        # We cannot document the closure as precise as PHPStan wants us to.
```

Comment:
Why not?

Reply:
If you find a nice way to do so, be my guest. The closure that we're passing here looks fine from my POV, so I'm not sure if this error is really worth fixing.